problems/2.functions.js

```diff
@@ -2,17 +2,21 @@
 * @param {string} name - The name of the user to greet
 * @returns {null}
 */
-function logGreeting() {
-
+function logGreeting(name) {
+  let greeting = "Hello";
+  let name = "Avery"
```

Comment:
Don't hard code Avery here. The name will be passed in by the person invoking the function.